### ISSUE TITLE
Errors in hie unit test suite involving cabal-helper since 04c2d34

### ISSUE THREAD

**Issue by jneira** [DanielG/cabal-helper#98]
* In the process of testing #93 and #97 in hie i got several errors running the hie unit test suite with
```cabal test :unit-test --test-options="--match CabalHelper"```
* All tests involving stack as build tool (but no cabal) fails with:
```
1) CabalHelper, cabal-helper spec, cradle discovery and loading, dummy filepath, finds none-cradle, stack repo
       uncaught exception: IOException of type NoSuchThing
       D:\ws\haskell\haskell-ide-engine\test\testdata\cabal-helper\simple-stack\.stack-work\dist\e626a42b\setup-config: openFile:
does not exist (No such file or directory)
```
* I've checked that effectively the `D:\ws\haskell\haskell-ide-engine\test\testdata\cabal-helper\simple-stack\.stack-work\dist` dir does not exist
* Then i've started to bisect cabal-helper commits since the hackage released version and i've found that the commit that triggers the error is 04c2d34 (1c0f2e8 works fine but it does not create the dist folders)

(NOTE: it would be great to have hackage releases/revisions tagged in github, to help diagnose those kind of bugs)

**Comment by jneira** [comment]
For completeness `git revert 04c2d34` makes the tests pass so the last commits after it look good

**Comment by DanielG** [comment]
Thanks for bisecting that, I must have pushed that commit by accident before finishing testing. Not sure how that happened..

**Comment by DanielG** [comment]
This was fixed by a18bbb2af92e9b4337e7f930cb80754f2408bcfd, and the commit is now part of #108.
